src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -216,6 +217,14 @@ public function channelBillingDataIs(
         $this->channelManager->flush();
     }
 
+    /**
+     * @Given the channel :channel is a "Mobile" type
```

Comment:
Maybe sth more generic?
```suggestion
     * @Given /^the (channel "[^"]+") is a (mobile|website|pos) type$/
```
and then you can add as a second argument `string $type`

The same you can do in other steps with channel type